GitOut/Features/Git/Log/GitTreeEvent.cs

```diff
@@ -37,6 +41,18 @@ public SolidColorBrush CommitBrush
 
         public IReadOnlyCollection<GitTreeNode> Nodes => nodes;
 
+        public bool IsSelected
+        {
+            get => isSelected;
+            set
+            {
+                if (isSelected == value) return;
```

Comment:
Optional: In many other classes we use a SetProperty method that wraps this behavior, in case we want to add that here as well?